xapian-applications/omega/configure.ac

```diff
@@ -246,6 +246,16 @@ AC_SEARCH_LIBS([socketpair], [xnet],
 dnl fnmatch() is used to match ignore pattern.
 AC_CHECK_FUNCS([fnmatch])
 
+dnl check if poppler is available.
+am_save_LIBS="$LIBS"
```

Comment:
I think `save_LIBS` would be a better name (which is the convention we use for saving and restoring variables in other `configure.ac` files).  I think `am_save_LIBS` is rather treading on automake's namespace (I notice `xapian-applications/omega/m4/iconv.m4` uses `am_save_LIBS`, but that's a third party file so we try to avoid modifying it if we can, so as to make importing new versions easier).

Reply:
Okay, I am going to fix it.